lib/builders/documentTypes/documentTypeCompositionBuilder.ts

```diff
@@ -2,7 +2,7 @@
 
 export class DocumentTypeCompositionBuilder {
```

Comment:
The build() method should return the object that has the following schema:
    {
      "documentType": {
        "id": "3fa85f64-5717-4562-b3fc-2c963f66afa6"
      },
      "compositionType": "Composition"
    }